### PR TITLE
data: ShivaRapolu01

### PR DESCRIPTION
**Pull request by ShivaRapolu01**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3132"></a>

### CODE REVIEW

File: data/ShivaRapolu01.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      "description": "All that hardward paid off. After getting shortlisted for many companies I got my first intern at Razorpay"
+    },
+    {
+      "title": "Social Media",
+      "date": "Dec 2022",
+      "icon": "DiOpensource",
+      "color": "blue",
+      "description": "I am trying to be active on my Twitter and LinkedIn profiles by providing valuable quality-rich useful content and sharing my learning in form of posts, tweets and threads. Hence Do follow me on both Twitter and LinkedIn"
+    }
+  ],
```
Comment (Njong392) [inline data/ShivaRapolu01.json:66]: Whoops. Looks like there is a trailing comma here 
```suggestion
  ]
```

File: data/ShivaRapolu01.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      "title": "Got Summer Intern at Razorpay",
+      "date": "2022",
+      "icon": "FaTrophy",
+      "color": "green",
+      "description": "All that hardward paid off. After getting shortlisted for many companies I got my first intern at Razorpay"
+    },
+    {
+      "title": "Social Media",
+      "date": "Dec 2022",
+      "icon": "DiOpensource",
```
Comment (Njong392) [inline data/ShivaRapolu01.json:62]: Unfortunately, I cannot find the icon `DiOpensource` in the docs. There is a section there that lists available icons.

**Review by Njong392 — CHANGES_REQUESTED** [review]
Thank you for contributing to LinkFree! :tada: Please take a look at the suggested changes I've made.

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thanks! 🎉

**Pull request merged**